### PR TITLE
Add license and coc

### PR DESCRIPTION
**Pull request by egekorkan**
I am adding these since I have added this to Netlify as well. As noted at https://github.com/w3c/wot-marketing/issues/304

The website is at https://wot-binding-templates.netlify.app/

**Note:** We are using netlify only for PR reviews

### CODE REVIEW

File: LICENSE copy.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+All Reports in this Repository are licensed by Contributors
```
Comment (danielpeintner) [inline LICENSE copy.md:1]: what is the right term?
* reports
* documents
* ?

File: LICENSE copy.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+under the
+[W3C Software and Document License](http://www.w3.org/Consortium/Legal/2015/copyright-software-and-document).
+
+Contributions to Specifications are made under the
+[W3C CLA](https://www.w3.org/community/about/agreements/cla/).
+
+Contributions to Test Suites are made under the
+[W3C 3-clause BSD License](https://www.w3.org/Consortium/Legal/2008/03-bsd-license.html)
```
Comment (danielpeintner) [inline LICENSE copy.md:9]: I am not sure whether those to items apply, do they?
I TD [LICENSE.md](https://github.com/w3c/wot-thing-description/blob/main/LICENSE.md) it is not there.

**Review by danielpeintner — CHANGES_REQUESTED** [review]
I think you used the wrong file name `LICENSE copy.md`

Some more comments below.

File: LICENSE copy.md (hunk 1 above)
Comment (egekorkan) [reply LICENSE copy.md:1]: Reports includes notes and recommendations. This is something I have copied from W3C and would not want to change anything

File: LICENSE copy.md (hunk 2 above)
Comment (egekorkan) [reply LICENSE copy.md:9]: Not sure why TD does not have this information but my guess would that its license is simply older than what W3C asks for now

**Comment by egekorkan** [comment]
@danielpeintner I have changed the filename, it was (obviously ^^) a copy paste mistake. The actual content of the license and CoC is something I do not want to change since they come from W3C

**Review by relu91 — APPROVED** [review]

**Review by danielpeintner — APPROVED** [review]

**Pull request merged**